DevoxxClientMobile/src/main/java/com/devoxx/views/ConfSelectorPresenter.java

```diff
@@ -90,18 +86,7 @@
     private final Comparator<Conference> pastConferenceComparator = (s1, s2) -> LocalDate.parse(s2.getFromDate()).compareTo(LocalDate.parse(s1.getFromDate()));
 
     public void initialize() {
-        final Optional<SettingsService> settingsService = Services.get(SettingsService.class);
-        if (settingsService.isPresent()) {
-            final SettingsService settings = settingsService.get();
-            final String eventType = settings.retrieve(DevoxxSettings.SAVED_CONFERENCE_TYPE);
-            if (eventType == null || eventType.equals("")) {
-                selector.setItems(service.retrieveConferences(Conference.Type.DEVOXX));
-            } else {
-                selector.setItems(service.retrieveConferences(Conference.Type.valueOf(eventType)));
-            }
-        } else {
-            selector.setItems(service.retrieveConferences(Conference.Type.DEVOXX));
-        }
+        selector.setItems(service.retrieveConferences(Conference.Type.DEVOXX));
```

Comment:
I haven't changed this because on my phone switching to Devoxx every time resulted in a black background behind the drawer. This was because of re-loading of Devoxx conferences when tapping on the Conf Selector item in the drawer.

Reply:
Not sure if I follow, but I tested it on desktop, and after the first time, the selector showed the last selection, not Devoxx. Do we need to fix that?

Reply:
Nope, we do not need to fix it. It is intended behaviour.